components/list/list-item-generic-layout.js

```diff
@@ -301,7 +302,9 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 			if (!listItem._tryFocus()) {
 				// ultimate fallback to generic method for getting next/previous focusable
 				const nextFocusable = previous ? getPreviousFocusable(listItem) : getNextFocusable(listItem);
-				if (nextFocusable) nextFocusable.focus();
+				if (nextFocusable && this._isContainedInSameList(curListItem, nextFocusable)) {
```

Comment:
Maybe we should name this `_isContainedInSameRootList` to make it clearer the scope of he check?

Reply:
Yep, I like that 👍